src/components/modeler/Modeler.vue

```diff
@@ -662,6 +662,18 @@ export default {
 
       this.paperManager.setPaperDimensions(clientWidth, clientHeight);
     },
+    keydownListener(event) {
+      const formElementHasFocus = event.target.toString().toLowerCase().indexOf('body') === -1;
```

Comment:
Do you think this would be more readable using [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes)?
```suggestion
      const formElementHasFocus = !event.target.toString().toLowerCase().includes('body');
```

Reply:
Definitely better yes